examples/nodejs/token-vending-machine/lambda/token-vending-machine/handler.ts

```diff
@@ -34,13 +34,13 @@ export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayPr
   }
 };
 
-interface VendedAuthToken {
-  authToken: string;
+interface VendedApiKey {
+  apiKey: string;
   expiresAt: number;
 }
 
-async function vendAuthToken(vendorAuthTokenSecretName: string, headers: APIGatewayProxyEventHeaders): Promise<VendedAuthToken> {
```

Comment:
We are vending disposable tokens, so we shouldn't rename this.

Reply:
you and @anitarua - great minds think alike! thanks for catching this

Reply:
fixed in https://github.com/momentohq/client-sdk-javascript/pull/877/commits/a0d8beddd9a32923612fd466851b9c00198832d7